### PR TITLE
pipeline: return the valid hep record

### PR DESCRIPTION
**Pull request by david-caro**
Signed-off-by: David Caro <david@dcaro.es>

### CODE REVIEW

File: hepcrawl/crawler2hep.py [hunk 1]
```diff
@@ -98,9 +98,9 @@ def _filter_affiliation(affiliations):
 
     try:
         builder.add_number_of_pages(
-            number_of_pages=int(crawler_record.get('page_nr'))
+            number_of_pages=int(crawler_record.get('page_nr', [])[0])
```
Comment (jacquerie) [inline hepcrawl/crawler2hep.py]: This looks scary, if `crawler_record` has no `page_nr` this blows up.

Comment (david-caro) [inline hepcrawl/crawler2hep.py]: yep, but it's catch right after (IndexError)

Comment (jacquerie) [inline hepcrawl/crawler2hep.py]: Whoops, didn't notice that.

File: setup.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
 install_requires = [
     'autosemver~=0.2',
-    'inspire-schemas~=30.0',
+    'inspire-schemas~=30.0,>=30.2.2',
```
Comment (kaplun) [inline setup.py]: Should you maybe put an upper bound?

Comment (david-caro) [inline setup.py]: the ~ is an upper bound for the major version.

Comment (kaplun) [inline setup.py]: Sorry, I haven't noticed it was still there the `~=30.0`

**Pull request merged**